docs/application/native/guides/app-management/dali-widget-app.md

```diff
@@ -0,0 +1,268 @@
+# DALi Widget Application
+
+
+You can create a widget application with DALi to display small version of application on the home screen.
```

Comment:
Please change "home screen" to "Home screen"

Reply:
Done